roles/download/defaults/main.yml

```diff
@@ -19,6 +19,7 @@ download_always_pull: False
 
 # Versions
 kube_version: v1.7.3
+kubeadm_version: "{{ kube_version }}"
```

Comment:
Why would kubeadm version differ from k8s version here?

Reply:
It defaults to match kube version, but it can be overridden if desired.

Reply:
ok, SGTM